### ISSUE TITLE
Do you use sync batch normalization internaly?

### ISSUE THREAD

**Issue by twmht** [hasanirtiza/Pedestron#46]
Hi,

I am new to mmdetection.

I found out that you set `ims_per_gpu` to 1. but the batch size should be grater than one if using batchnorm in training mode in pytorch.

Do you use synchronous batch normalization internally? I can' find the related code.
